src/commandHandler.ts

```diff
@@ -205,6 +202,7 @@ async function handleUser(text : string, outputs : string[], errorMessage : stri
 
 // get a single input
 async function getUserInput(prompt:string): Promise<string | undefined> {
+	// though flagged as a code smell, this must be an await or we don't get the user input
```

Comment:
isn't the problem that the methods that are calling the getUserInput that should have an await?

Reply:
Look at line 215 in the same file in `collectUserInput` - we ARE using await. And that's the only place it's being used.

Reply:
Testing manually I found that not having the await does not properly pause long enough to retrieve the user input, thus the note about the code smell.

Reply:
Can you provide steps to reproduce when it is not working?
it has the same behavior with and without for me.

Working when typing directly and keeping focus on the input box, displaying `Didact was unable to collect user input: Error: Input aborted` notification when clickign outside of the input box.

Reply:
Since I am unable to reproduce the behavior I saw the day I added that comment, I will remove it and the extra await. It seems to work fine.